src/lib/strategies/GlobalKeyEventStrategy.js

```diff
@@ -73,6 +73,8 @@ class GlobalKeyEventStrategy extends AbstractKeyEventStrategy {
 
     this._updateDocumentHandlers();
 
+    this._initHandlerResolutionState();
```

Comment:
⬜️⚔️Check the behaviour of `this._updateDocumentHandlers()` to make sure calling these methods in this sequence will always work as intended.

Reply:
⬜️ 🚀See if there is a way of wrapping the resetting of state in `this._initHandlerResolutionState()` in a condition that will abort resetting the state if it's apparent handler resolution hasn't actually taken place yet, and the state is already set to the default values.